plugin/cmd/kubectl/utils/ui.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import "fmt"
+
+func Green(str string) string {
+	return fmt.Sprintf("\x1b[32;1m%s\x1b[0m", str)
```

Comment:
Note that these aren't supported and will look bad on most Windows terminals. One of the major reasons we didn't add broader support to colors in `kubectl` yet. If you absolutely want them, I suggest using a library like [this one](https://github.com/fatih/color).